tests_tf/test_attacks.py

```diff
@@ -927,14 +814,44 @@ def test_generate_np_gives_clipped_adversarial_examples(self):
     self.assertTrue(np.max(x_adv) < .301)
 
 
-class TestMadryEtAl(CleverHansTest):
+class TestProjectedGradientDescent(CommonAttackProperties):
   def setUp(self):
-    super(TestMadryEtAl, self).setUp()
+    super(TestProjectedGradientDescent, self).setUp()
 
     self.sess = tf.Session()
     self.model = SimpleModel()
-    self.attack = MadryEtAl(self.model, sess=self.sess)
+    self.attack = ProjectedGradientDescent(self.model, sess=self.sess)
+
+  def test_generate_np_gives_adversarial_example_linfinity(self):
```

Comment:
I think these methods here should be in TestProjectedGradientDescent above

Reply:
Done